jest-common/src/main/java/io/searchbox/core/Cat.java

```diff
@@ -0,0 +1,92 @@
+package io.searchbox.core;
+
+import com.google.gson.Gson;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParser;
+import io.searchbox.action.AbstractAction;
+import io.searchbox.action.AbstractMultiIndexActionBuilder;
+import io.searchbox.action.AbstractMultiTypeActionBuilder;
+import org.apache.commons.lang3.builder.EqualsBuilder;
+
+/**
+ * @author Bartosz Polnik
+ */
+public class Cat extends AbstractAction<CatResult> {
+    private final static String PATH_TO_RESULT = "result";
+    public Cat(AbstractAction.Builder<Cat, ? extends Builder> builder) {
+        super(builder);
+    }
+
+    Cat() {
+    }
+
+    @Override
+    public String getRestMethodName() {
+        return "GET";
+    }
+
+    @Override
+    public String getPathToResult() {
+        return PATH_TO_RESULT;
+    }
+
+    @Override
+    public CatResult createNewElasticSearchResult(String responseBody, int statusCode, String reasonPhrase, Gson gson) {
+        return createNewElasticSearchResult(new CatResult(gson), responseBody, statusCode, reasonPhrase, gson);
+    }
+
+    @Override
+    protected JsonObject parseResponseBody(String responseBody) {
+        JsonObject result = new JsonObject();
+        if (responseBody != null && !responseBody.trim().isEmpty()) {
+            result.add(PATH_TO_RESULT, new JsonParser().parse(responseBody).getAsJsonArray());
+        }
+
+        return result;
+    }
+
+    @Override
+    public boolean equals(Object obj) {
```

Comment:
There's no need for that as `AbstractAction.equals` already compares complete URI.